vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html

```diff
@@ -970,6 +970,40 @@
        */
       then: function(callback) {
         return this._grid.then(callback);
+      },
+
+      /**
+       * The row detail generator that is used for generating detail content for rows.
+       *
+       * Example:
+       * ```
+       * grid.rowDetails.detailsGenerator = function(rowIndex) {
+       *  var detail = document.createElement('div');
+       *  detail.innerText = 'Row detail content for row ' + rowIndex;
```

Comment:
`element.innerText` is only supported in some browsers, it’s not a standard property. Use `element.textContent` instead.